lookout/core/sqla_model_repository.py

```diff
@@ -66,11 +67,22 @@ def __init__(self, db_endpoint: str, fs_root: str, max_cache_mem: int, ttl: int,
         :param engine_kwargs: Passed directly to SQLAlchemy's `create_engine()`.
         """
         self.fs_root = fs_root
+        db_endpoint_components = urlparse(db_endpoint)
+        if db_endpoint_components.password is not None:
```

Comment:
would be great to have a comment which explains why we are doing what we are doing here